lib/tasks/demo.js

```diff
@@ -199,8 +193,9 @@ function hasUniqueNames(demos) {
 
 function updateOrigamiJsonDemoList(demosBuilt) {
 	var origamiDemoConfig = demosBuilt.map(function(conf) {
+
 		return {
-			path: "/" + conf.outputHtmlPath,
+			path: '/demos/' + conf.name + '.html',
```

Comment:
Though it's not really that important because I just want to generate the string, that path isn't going to be used anywhere